### ISSUE TITLE
Sync not working

### ISSUE THREAD

**Issue by BirchBranch2121** [futo-org/grayjay-android#1640]
### What happened?

I have just downloaded the new GrayJay desktop app and am hyped! Unfortunately, I can't sync my settings with the QR code since the screen after the QR scan screen doesn't do anything. A code is in the text input box, but nothing happens when the blue button at the bottom of the screen is pressed. 

### Grayjay Version

276

### What plugins are you seeing the problem on?

All

### When do you experience the issue?

- [ ] While logged in
- [ ] While logged out
- [ ] N/A

### Relevant log output

```shell
N/a
```

**Comment by codebymatter** [comment]
I had an issue with Sync at first and what I did ia unlink and did a re-link.

Nothing hapoened still so I closed Grayjay desktop and upon opening the apo, it works. All my settings and subscriptions from YouTube and Rumble are on the Subscription tab like on Android app as well as Playlista and Ratings.  I would go on but the Desktoo app issues belong in [this](https://github.com/futo-org/Grayjay.Desktop/issues) girhub repo.

I also turned off sync on desktop and Android apps to avoid something like 2160p and 1440p defaults res on Desktop fapp from overriding 720p and 1080p defaults on Android app. I don't know if having aync on both apps will actually change setfings and such to reflect both.

**Comment by Zvonimir-FUTO** [comment]
@BirchBranch2121 could you please clarify from which device are you starting a sync? I'm presuming that you are scanning the QR from desktop on your phone. Afterwards, you should see a small popup on desktop version asking you to accept the sync from where the sync starts.

**Comment by BirchBranch2121** [comment]
@Zvonimir-FUTO Yeah I just tried it, I was scanning the QR code on my laptop from my phone but they were both on VPNs... once I turned that off it ended up syncing just fine.

**Comment by Zvonimir-FUTO** [comment]
Yeah sync with proxying is not yet implemented but it's in progress. Currently it will only work if you are on the same network without VPN but we are working on improving this so please be patient.
